src/project-management/AppMetadata.ts

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface AppMetadata {
+  readonly appId: string;
+  readonly displayName: string;
+  readonly platform: AppPlatform;
+}
+
+export enum AppPlatform {
```

Comment:
Nit: Lets move this to the top, so the parent and child classes are arranged next to each other.